code/modules/surgery/tools.dm

```diff
@@ -32,6 +35,9 @@
 	origin_tech = "materials=1;biotech=1"
 	attack_verb = list("attacked", "pinched")
 
+/obj/item/hemostat/suicide_act(mob/living/user)
+ 	user.visible_message("<span class='suicide'>[user] tears [user.p_them()]self apart with a snip, a cut, and a [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")

```

Comment:
inconsistent indentation here as well

Reply:
p_them doesn't ever return "them" does it because themself is weird.

Reply:
https://github.com/tgstation/tgstation/blob/551ede6d35bc955fa83979049441321f46ab9653/code/__HELPERS/pronouns.dm#L67

it does.